Examples/Io/NuclearInteractions/src/detail/NuclearInteractionParametrisation.cpp

```diff
@@ -351,7 +351,7 @@ TVectorF softProbability(const EventCollection& events) {
   }
 
   TVectorF result(1);
-  result[0] = counter / (float)events.size();
+  result[0] = counter / events.size();
```

Comment:
`float counter` looks a bit odd

Reply:
indeed, I changed the counter to `size_t` and cast later to `float`